STEERING.md

```diff
@@ -24,7 +24,8 @@ The Steering Committee’s responsibilities are to:
 The Steering Committee comprises at most 7 people. To be eligible for
 membership in the Steering Committee, you must:
 
-1. Be responsible for a production Linkerd deployment of non-trivial size
+1. Have been responsible for a production Linkerd deployment of non-trivial
```

Comment:
If we're going to go in this direction, for clarity we might want to make this "Be currently responsible for a production Linkerd deployment of non-trivial size, or have been responsible for one within the past two years."

Reply:
👍 Updated!